README.md

```diff
@@ -16,7 +16,10 @@ import { Expo } from 'expo-server-sdk';
 
 // Create a new Expo SDK client
 // optionally providing an access token if you have enabled push security
-let expo = new Expo({ accessToken: process.env.EXPO_ACCESS_TOKEN });
+let expo = new Expo({
+  accessToken: process.env.EXPO_ACCESS_TOKEN,
+  useFCM: false // this can be set to true in order to use the FCM v1 API
```

Comment:
```suggestion
  useFCMv1: false // this can be set to true in order to use the FCM v1 API
```

Reply:
You're succesting we should use useFCMv1 as prop? Because that also means I should change it everywhere else, and not just here.